### PR TITLE
Prepare package

### PR DESCRIPTION
**Pull request by kou**

### CODE REVIEW

File: substrait.gemspec [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# SPDX-License-Identifier: Apache-2.0
+
+require_relative "lib/substrait/version"
+
+Gem::Specification.new do |spec|
+  spec.name = "substrait"
+  spec.version = Substrait::VERSION
+  spec.homepage = "https://substrait.io/"
+  spec.authors = ["Sutou Kouhei"]
+  spec.email = ["kou@clear-code.com"]
```
Comment (kou) [inline substrait.gemspec:12]: Is there any preferred author?
For example, Apache Arrow uses "Apache Arrow Developers" and "dev@arrow.apache.org": https://github.com/apache/arrow/blob/master/ruby/red-arrow/red-arrow.gemspec#L32-L33

**Comment by kou** [comment]
@jacques-n @cpcloud Could you review this?

CI result: https://github.com/kou/substrait-ruby/actions/runs/2344242216

File: substrait.gemspec (hunk 1 above)
Comment (cpcloud) [reply substrait.gemspec:12]: Up to you. You can put `Substrait developers` or leave it as is.

**Review by cpcloud — APPROVED** [review]

File: substrait.gemspec (hunk 1 above)
Comment (kou) [reply substrait.gemspec:12]: OK. I leave this as is because I want to maintain this.

**Review by cpcloud — APPROVED** [review]

**Comment by kou** [comment]
@cpcloud Thanks for merging this!

I want to release this to https://rubygems.org/ . Could you give me "Admin" or "Write" permission of this repository so that I can tag, enable GitHub Pages and so on?

**Pull request merged**